configuration.go

```diff
@@ -57,8 +58,11 @@ func (conf Configuration) Initialize() (Configuration, error) {
 	}
 
 	conf.URL = conf.AdminURL()
-	if conf.ThemeID != 0 {
-		conf.URL = fmt.Sprintf("%s/themes/%d", conf.URL, conf.ThemeID)
+
+	if themeID, err := strconv.ParseInt(conf.ThemeID, 10, 64); err == nil {
+		conf.URL = fmt.Sprintf("%s/themes/%d", conf.URL, themeID)
+	} else {
+		return conf, fmt.Errorf("missing theme_id")
```

Comment:
Here it might be useful to actually report if the theme is actually in an invalid format by appending the error message?